### DISCUSSION TITLE
It would be nice if variables could be used in the README.md

### DISCUSSION THREAD

**Discussion by justinmchase** [denoland/deno#15961] [Ideas]
In most / all readme's there is usually a little usage example to get someone started. It will usually start with a line that shows how to import the current module. That import statement usually should have a version number in it. Trying to keep that version number up to date with the actual latest version tag isn't super hard but it does require a little automation, or dilligence, or both.

What would be nice is if deno.land/x recognized, at least, the $GIT_TAG variable and just did a straight replacement, so if I had some snippet in my readme such as: like:

````ts
```ts
import { example } from "https://deno.land/x/example@$GIT_TAG/mod.ts";
```
````

Then it would render like this, when viewed relative to the tag named `0.1.0` in the deno land third party module UI.
```ts
import { example } from "https://deno.land/x/example@0.1.0/mod.ts";
```

**Comment by kitsonk** [comment]
This is a suggestion for dotland: https://github.com/denoland/dotland/issues/2494
